.github/workflows/release.yml

```diff
@@ -27,28 +27,27 @@ jobs:
         with:
           environment-name: build-env
           create-args: >-
-            python={{min_python}}
             conda-build
             boa
-      - run: conda mambabuild --channel conda-forge --channel scipp --python={{min_python}} --no-anaconda-upload --override-channels --output-folder conda/package conda
+      - run: conda mambabuild --channel conda-forge --channel scipp --no-anaconda-upload --override-channels --output-folder conda/package conda
```

Comment:
Since we use the template only for `noarch` packages I think this is not required?